src/app/pages/camper/schedule/schedule.component.ts

```diff
@@ -1,15 +1,75 @@
 import { Component, OnInit } from '@angular/core';
+import { ScheduleService } from "../../../service/schedule.service";
+import { interval } from 'rxjs';
 
 @Component({
   selector: 'app-schedule',
   templateUrl: './schedule.component.html',
   styleUrls: ['./schedule.component.scss']
 })
 export class ScheduleComponent implements OnInit {
+	
+  	private todayDate;
+  	private schedulelist;
+  	private hours;
+  	private mins;
+  	private secs;
 
-  constructor() { }
+  constructor(
+  	public schService:ScheduleService
+  	) { }
 
   ngOnInit() {
+  	this.schService.getSheet().subscribe(response => {
+
+	  	var date = ["20 JULY", "21 JULY", "22 JULY"];
```

Comment:
I understand that this is a dummy function, like you're still trying out. 

But it is still good to abstract or refractor whatever you can. 

i.e. do you think this date array is gonna be declared in this file itself? if yes, maybe can put it in a variable on a higher scope. Or if you think we are gonna obtain this set of dates from somewhere, maybe convert it into a function call. 

i.e. 
```javascript
class ScheduleComponent implements OnInit {
     private const CAMPDAYS = ["20 JULY", "21 JULY", "22 JULY"];
     ....
}

OR 

function getCampDates() {
    return ["20 JULY", "21 JULY", "22 JULY"];
}
```